storage/store.go

```diff
@@ -909,6 +909,15 @@ func (s *Store) BootstrapRange() error {
 	if err := engine.MVCCPutProto(batch, ms, key, now, nil, zoneConfig); err != nil {
 		return err
 	}
+
+	// We reserve the first 1000 descriptor IDs.
+	key = keys.DescIDGenerator
```

Comment:
Added a test into `TestCreateTable` that the newly created table has a non-reserved ID.